helpers/install_docker.sh

```
@@ -2,32 +2,54 @@
-echo "🚦 Enabling and starting Docker service..."
-sudo systemctl enable docker
-sudo systemctl restart docker
+echo "� Setting up Docker rootless mode..."
+# Install uidmap package for rootless mode
+sudo apt-get install -y uidmap dbus-user-session
+
+# Install Docker rootless setup tool
+dockerd-rootless-setuptool.sh install
+
+# Enable systemd lingering for the current user
+sudo loginctl enable-linger $(whoami)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)